doc/classes/Generic6DOFJoint.xml

```diff
@@ -5,6 +5,8 @@
 	</brief_description>
 	<description>
 		The first 3 DOF axes are linear axes, which represent translation of Bodies, and the latter 3 DOF axes represent the angular motion. Each axis can be either locked, or limited.
+		
```

Comment:
Blank lines in the class reference are not necessary.
```suggestion
```

Reply:
Yeah, I thought I already got rid of it